odl/util/utility.py

```diff
@@ -72,12 +78,80 @@ def indent(string, indent_str='    '):
 
     Indenting by random stuff:
 
-    >>> print(indent(text, indent_str='|----|'))
-    |----|This is line 1.
-    |----|Next line.
-    |----|And another one.
+    >>> print(indent(text, indent_str='<->'))
+    <->This is line 1.
+    <->Next line.
+    <->And another one.
     """
-    return '\n'.join(indent_str + row for row in string.split('\n'))
+    return '\n'.join(indent_str + row for row in string.splitlines())
+
+
+def dedent(string, indent_str='   ', max_levels=None):
+    """Revert the effect of indentation.
```

Comment:
`IndentationOperator.inverse`!

(no, I'm not serious).

Reply:
:scream: